backend/fms_core/viewsets/full_sample.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from fms_core.models import FullSample
+from fms_core.serializers import FullSampleSerializer
+
+from ._utils import _list_keys
+from ._constants import _full_sample_filterset_fields
+
+class FullSampleViewSet(viewsets.ModelViewSet):
+    queryset = FullSample.objects.select_related("individual", "container", "sample_kind", "biosample", "derived_sample").all().distinct()
```

Comment:
What FullSample attribute is the distinct performed on ? The FullSample id?

Reply:
I think distinct here is superfluous, I used it when I was trying to do prefetch_related() but now we don't need it anymore.